.github/workflows/actions.yml

```diff
@@ -0,0 +1,114 @@
+name: innovativeproject-loancalculator
+
+on:
+  push:
+    branches: '**'
```

Comment:
Seems like this line and line 7 are not needed in order to trigger github-actions:
Please see example actions defined in starter-workflows repo:
https://github.com/actions/starter-workflows/blob/main/.github/workflows/validate-data.yaml